### PR TITLE
Rebuild install script generator

### PR DESCRIPTION
**Pull request by AlecRust**
### Current problem

Couple things about the [current install script generator](https://hestiacp.com/install.html) that aren't ideal:
- It's built with Vue.js yet doesn't seem to be employing the benefits of such a library. Namely the user configures the options they want and then click "Submit" as if there is some processing required - but "Submit" just opens a modal. If the user wants to change the command they have to close the modal, change the options and "submit" again.
- If the user changes no options and just clicks "Submit", the installation command that is provided is long and includes a number of options, unlike the basic [documented install command](https://hestiacp.com/docs/introduction/getting-started.html#step-2-run). This is because all the _default_ options are included in the command, despite them being the default.

### Proposed solution

This rebuilds the install script generator from scratch to address the above issues and generally simplify the component. The install command is always visible at the top of the page and updates as the user changes the options. The command is just `bash hst-install.sh` by default, and options are only included in the command if the value differs from the default.

### Demo

[**Try it out**](https://docs-rebuild-install-generat.hestiacp.pages.dev/install.html)

### Screenshots

https://github.com/hestiacp/hestiacp/assets/247634/44a650fa-2ca2-42e2-9135-7a2f1ec94354

### Notes
- MultiPHP is [documented](https://hestiacp.com/docs/introduction/getting-started.html#list-of-installation-options) to not be enabled by default, but the current install command generator lists it as enabled by default. I have updated the "List of installation options" documentation to indicate this is in fact enabled by default.
- The [current install command generator](https://hestiacp.com/install.html) provides `sudo bash hst-install.sh` but the [main docs](https://hestiacp.com/docs/introduction/getting-started.html#step-2-run) suggest `bash hst-install.sh` without "sudo". The new install script generator omits "sudo" to match.
- With a new `label` property the options are presented to the user in a more friendly way using their the full brand name with appropriate casing. Users can now click anywhere on the option "card" to toggle it.

### CODE REVIEW

File: docs/_data/options.js [hunk 1]
```diff
@@ -1,178 +1,201 @@
...
+		flag: 'apache',
+		label: 'Apache',
+		description: 'Web server with htaccess support.',
+		default: 'yes',
+	},
+	{
+		flag: 'phpfpm',
+		label: 'PHP-FPM',
+		description: 'FastCGI Process Manager for executing PHP scripts.',
+		default: 'yes',
```
Comment (jaapmarcus) [inline docs/_data/options.js]: I am sure the default option is no

https://github.com/hestiacp/hestiacp/blob/4153e6c0ff05057610ed6d62fc788f2ef664eb0f/install/hst-install-ubuntu.sh#L58C18-L58C18

File: docs/_data/options.js [hunk 2]
```diff
@@ -1,178 +1,201 @@
...
 	},
 	{
-		name: ' --clamav',
-		id: 'clamav',
-		param: '--clamav',
-		desc: 'Scans your email inbox for viruses.',
-		selected: true,
-		depends: 'exim',
+		flag: 'iptables',
+		label: 'iptables',
```
Comment (jaapmarcus) [inline docs/_data/options.js:167]: I assume you used the default "spelling"  it looks odd but

File: docs/_data/options.js (hunk 1 above)
Comment (AlecRust) [reply docs/_data/options.js]: Looks like this is wrong on the current install command generator. Fixed in 9e0e9caae.

File: docs/_data/options.js (hunk 2 above)
Comment (AlecRust) [reply docs/_data/options.js:167]: Yeah `iptables` seems to be the correct casing 🤷

**Review by jaapmarcus — APPROVED** [review]

**Pull request merged**